hardwared/serial.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * This program sends the string from command line argument argv[1] to the serial port
+ * If there are no arguments - sends "Nothing"
+ * Can be used to debug communication between hardware device and our hsmd daemon
+ * TODO: add serial_read function
+ */
+
+#include <stdio.h>
+#include <fcntl.h>   /* File Control Definitions           */
+#include <termios.h> /* POSIX Terminal Control Definitions */
+#include <unistd.h>  /* UNIX Standard Definitions      */
+#include <errno.h>   /* ERROR Number Definitions           */
+#include <string.h>
+
+// write here the serial port you have Arduino plugged to
+// should be autodetected afterwards
+static char serialport[] = "/dev/cu.usbmodem1461";
+
```

Comment:
This seems to be different than the serial port device that appears on my machine (Arch Linux).. for me, `/dev/ttyACM1` appears (as well as `/dev/usb/hiddev1`). If we can autodetect this it would be best, as your comment says, otherwise maybe we need to check a few common paths..